src/Api/ProjectsApi.php

```diff
@@ -100,4 +143,15 @@ public function projectsUserUserIdGet($userId, $maxVersion = null, $limit = 20,
     $responseCode = Response::HTTP_NOT_IMPLEMENTED;
     // TODO: Implement projectsUserUserIdGet() method.
   }
+
+  /**
+   * @param Program[] $programs
+   *
+   * @return array
+   */
+  public function getWantedInformation($programs)
```

Comment:
not a very meaningful method name +
no need for this method, just use the correct ResponseClasses directly in the method.